packages/labs/ssr-dom-shim/src/index.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const attributes: WeakMap<HTMLElement, Map<string, string>> = new WeakMap();
```

Comment:
What are our Node version requirements? It would be nice to just use real private fields. Otherwise, we use `_shadowRoot` for the shadow root, I don't see why we wouldn't be consistent and use `_attributes_` for attributes.

Reply:
We still support Node 14